crates/e2e/macro/src/config.rs

```diff
@@ -128,18 +136,59 @@ mod tests {
     }
 
     #[test]
-    fn duplicate_args_fails() {
+    fn duplicate_additional_contracts_fails() {
         assert_try_from(
             syn::parse_quote! {
                 additional_contracts = "adder/Cargo.toml",
                 additional_contracts = "adder/Cargo.toml",
             },
             Err(
-                "encountered duplicate ink! e2e test `additional_contracts` configuration argument",
+                "encountered duplicate ink! E2E test `additional_contracts` configuration argument",
+            ),
+        );
+    }
+
+    #[test]
+    fn duplicate_environment_fails() {
```

Comment:
Would also be cool to see some UI tests for the E2E macros, but that would be for a future PR